### ISSUE TITLE
INFO:tensorflow:global step 529: loss = 28296075926372180033536.0000 (2.561 sec/step)

### ISSUE THREAD

**Issue by navinkeshava** [tensorflow/models#4855]
i am getting this kind of error why is this i am really not able to analyze

**Comment by xDooooot** [comment]
Hi.
I soved this by checking my training data. There was something wrong with my xml files. That's work for me. Hoping it will work for you :)

**Comment by navinkeshava** [comment]
thanks for your reply
i checked your solution and tried also but i am not able find any error in
xml.  it would be great if you brief it  what type of  error in xml files

thank you

**Comment by karmel** [comment]
I apologize, but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please fill out the issue template (https://github.com/tensorflow/tensorflow/issues/new). Please provide all the information it asks. Thank you.

**Comment by xDooooot** [comment]
Hi.
I messed up the correspondence of jpgs and xmls. I mean, worng xml for a single jpg file.
Good luck
@navinkeshava

**Comment by navinkeshava** [comment]
thanks a lot for your quick reply , it helped me to figure out the error .

**Comment by xDooooot** [comment]
My pleasure.   :)
@navinkeshava

**Comment by wt-huang** [comment]
Closing as this is resolved
